packages/storefront-lwc/src/modules/commerce/basket/basket.js

```diff
@@ -51,21 +57,14 @@ export default class Basket extends LightningElement {
         return this.basket.selectedShippingMethodId;
     }
 
-    connectedCallback() {
-        ShoppingBasket.getCurrentBasket()
-            .then(basket => {
-                this.basket = basket;
-                this.products = basket.products ? basket.products : [];
-                this.loading = false;
-            })
-            .catch(error => {
-                console.log('error received ', error);
-            });
-    }
+    @wire(useMutation, {
+        mutation: REMOVE_ITEM_FROM_BASKET,
+        lazy: true,
+    })
+    removeItemFromBasket;
 
     updateBasket(event) {
         this.basket = { ...this.basket, ...event.detail.updatedBasket };
-        this.products = event.detail.updatedBasket.products;
-        ShoppingBasket.basket = this.basket;
+        //this.products = event.detail.updatedBasket.products;
```

Comment:
We should remove the commented out code.

Reply:
my bad, missed this one. Will fix it, good catch. thanks.